README.md

```diff
@@ -14,10 +14,13 @@ You need a `~/.jump.json` config file that looks like this:
 {
   "config": {
     "github_access_token": "youraccesstokenhere",
-    "globalShortcut": "Command+J"
+    "globalShortcut": "Command+J",
+    "host": "optional",
+    "pathPrefix": "optional"    
```

Comment:
Could we put these under "githubEnterprise" to make it a little more explicit what it is for?

```json
"githubEnterprise": {
  "host": "optional",
  "pathPrefix": "optional"
}
```